File: openapi/code_samples/javascript/api_company_sector_{caen}_top.js
const express = require('express');
const app = express();
const port = 3000;

app.get('/api/company/top-10-companies/:caen', (req, res) => {
    const caen = req.params.caen;
    // Your implementation here
    res.status(200).json({});
});

app.listen(port, () => {
    console.log(`Server is listening at http://localhost:${port}`);
});

File: openapi/code_samples/csharp/api_company_sector_{caen}_top.cs
using Microsoft.AspNetCore.Mvc;
using Swashbuckle.AspNetCore.Annotations;

[ApiController]
[Route("api/[controller]")]
public class CompanyController : ControllerBase
{
    [HttpGet]
    [Route("top-10-companies/{caen}")]
    [SwaggerOperation(Summary = "Retrieves the top 10 companies on a given sector CAEN.")]
    [ProducesResponseType(typeof(TopCompaniesByFiscalValueView[]), 200)]
    [ProducesResponseType(typeof(Fault), 400)]
    public IActionResult GetTop10CompaniesByCaen([FromRoute] string caen)
    {
        // Your implementation here
        return Ok();
    }
}
